core/src/test/java/org/apache/iceberg/TestMetrics.java

```diff
@@ -114,6 +114,56 @@ public abstract File writeRecords(Schema schema, Map<String, String> properties,
 
   public abstract int splitCount(File parquetFile) throws IOException;
 
+  @Test
+  public void testMetricsForRepeatedValues() throws IOException {
```

Comment:
What do you want to test with this case?

Reply:
This test validates the contract that counts include repeated values. In my ORC implementation counts were not included and there was no test validating that contract that now I've documented.